modules/howtos/pages/kv-operations.adoc

```diff
@@ -209,11 +209,7 @@ You can set an expiry value from a `Duration` when creating a document:
 include::example$KvOperations.java[tag=expiry-insert]
 ----
 
-We recommend using a relative duration only if the provided value is less than 30 days. 
-The reason is that the server will assume any value larger than that to be an absolute Unix timestamp. 
-The SDK tries its best to coerce it into sane values, but to avoid any unexpected behavior please stick with relative duration for durations of less than 30 days.
-
-For longer expiry values, use an `Instant` instead:
+Alternatively, you can specify the expiry as an `Instant`:
```

Comment:
Does that mean that the "magic" behaviour above has been removed?
e.g. any numeric value is a duration, and only an `Instant` object represents a point in time?

Reply:
The expiry parameter is strongly typed as either a `Duration` or an `Instant`. The change is that starting with Java SDK 3.0.6, a Duration longer than 30 days is does what the user expects, instead of causing the document to expire immediately.

Incidentally, I didn't realize this PR removed the blank spaces at the ends of lines everywhere. That was not my intent; I'll update it so it only removes this paragraph.